cmake/3rdParty/Platform/Linux/OpenSSL_linux.cmake

```diff
@@ -0,0 +1,19 @@
+#
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of this distribution.
+#
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+#
+
+# Use system default OpenSSL library instead of maintaining an O3DE version for Linux
+find_package(OpenSSL)
+if (NOT OpenSSL_FOUND)
+   message(FATAL_ERROR "Compiling on linux requires the development headers for OpenSSL.  Try using your package manager to install the OpenSSL development libraries following https://wiki.openssl.org/index.php/Libssl_API")
+else()
+    # OpenSSL targets should be considered as provided by the system
+    set_target_properties(OpenSSL::SSL OpenSSL::Crypto PROPERTIES LY_SYSTEM_LIBRARY TRUE)
+
```

Comment:
If this ever causes warnings and include warnings, we may have to update our library to use ly_system_include stuff.  We also may have to do import configuration mapping.
If there's no problem with that, we can wait...

Reply:
I didn't get warnings during my local testing and the build worked fine. Will verify on Jenkins as well once the infra team helps to set up a new node which include the libssl-dev package.